src/components/AeButton.vue

```diff
@@ -51,5 +51,9 @@ export default {
   span {
     vertical-align: inherit;
   }
+
+  &.green {
```

Comment:
I would switch this class by a boolean property instead because seeing it's usage it can be mixed up with:
- a class to add styles locally (maybe accidentally removed on refactoring as unused)
- some global class that makes anything green

Reply:
just currently unused, will be used in word bazaar

is local to that class so should be fine from a global view

Reply:
I mean, it is unclear when it is used like:
```vue
<template>
  <AeButton class="green" />
</template>

<style scoped>
.red {
  color: red;
}
</style>
```
- is green defined globally?
- is green had corresponding styles defined in this component before?
- is green defined inside AeButton?

Using property it becomes obvious that `green` is defined on the side of AeButton:
```vue
<template>
  <AeButton green />
</template>
```

> just currently unused, will be used in word bazaar

it may look unused like the definition is missed like in the example above

> is local to that class so should be fine from a global view

it doesn't pollute the global scope, but its usage may look like it is defined global as in the example above

After all, it is not consistent with some other components like [Loading](https://github.com/aeternity/superhero-ui/pull/901/files#diff-7fb57159f993f8d56e574f7be6ea421b0cdb807a44881a10ea4022d352061f04R12).

Reply:
Sorry I didn't understand earlier, thanks for the explanation, adjusted here https://github.com/aeternity/superhero-ui/pull/901/commits/6fc8159178229c11a2ef09c9a13655be5e828bcd